configure/files/container.rb

```diff
@@ -1,8 +1,9 @@
 Ohai.plugin(:Container) do
+  depends "etc/passwd"
   provides "container"
 
   collect_data(:default) do
-    if File.exist?("/.dockerenv") || !!(File.read("/proc/1/cgroup") =~ %r[^\d+:\w+:/(lxc|docker)/])
+    if File.exist?("/.dockerenv") || !!(File.read("/proc/1/cgroup") =~ %r[^\d+:\w+:/(lxc|docker)/]) || etc["passwd"].key?("gitpod")
```

Comment:
Do we need to keep things here specifically for `gitpod` as a service?

Reply:
Yeah gitpod evades the other 2 docker checks